object/map.js

```diff
@@ -0,0 +1,11 @@
+
+if (!Object.prototype.map) {
+  Object.defineProperty(Object.prototype, 'map', {
+    value: function(callback) {
+      var object = this
```

Comment:
Hmm I don't see how it would but I can
On Fri, Sep 18, 2015 at 12:09 Darin Haener <notifications@github.com> wrote:

> In object/map.js
> <https://github.com/Legitcode/rubyfill/pull/4#discussion_r39890414>:
>
> > @@ -0,0 +1,11 @@
> > +
> > +if (!Object.prototype.map) {
> > +  Object.defineProperty(Object.prototype, 'map', {
> > +    value: function(callback) {
> > +      var object = this
>
> Right. I'm wondering. Should we include a test that ensures the original
> value of the object doesn't get mutated? I don't think that's desired
> behavior.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Legitcode/rubyfill/pull/4/files#r39890414>.
>